Alignment/CTPPS/plugins/CTPPSFastLocalSimulation.cc

```diff
@@ -3,7 +3,7 @@
 * This is a part of TOTEM offline software.
 * Authors: 
 *  Jan Kašpar (jan.kaspar@gmail.com) 
-*    
+*  Cristian Baldenegro (crisx.baldenegro@gmail.com)
```

Comment:
Nothing should be changed to `CTPPSFastLocalSimulation` - could you please revert your changes? Instead, we will add a module to the processing sequence with will convert rechits to UVPatterns.